### DISCUSSION TITLE
Status: draft or status: hidden does not work properly

### DISCUSSION THREAD

**Discussion by learningchannel** [datenstrom/community#207] [Report a bug] [answered]
hi -- is this normal for yellow to show "hidden pages" when /edit/ is applied to the url?

example: www.website.com/food <--this page has a `status: draft` or `status: hidden` or `status:maintenance` with the plugin. When logged out, if i goto the url i am not able to view the contents--i get an error, or a draft status message. if however I visit www.website.com/edit/food <-- i get the login screen but i am totally able to view/see loaded contents of the page behind the login screen.

**Comment by markseuffert** [comment]
Looks like a bug.

**Comment by markseuffert** [comment, marked as answer]
Good that you found it. The webinterface plugin doesn't handle the status if no user is logged in. Unfortunately it forgot to tell the core to handle the status, that's why it was ignored. Should be fixed now. Please update your webinterface plugin and try again.

**Comment by learningchannel** [comment]
good good! new plugin works perfectly. thank you for the quick fix.

**Comment by markseuffert** [comment]
Thanks a lot for testing. Let us know if you find more bugs.
